### PR TITLE
Using also-after-suspend flag to keep test consistency before and after suspend for 22.04 automated test plan

### PR DESCRIPTION
**Pull request by weizhenwu**
## Description

To make sure we test extact same cases before and after suspend, I've made following change:
1. Use flag: also-after-suspend for all test cases
2. Modify desktop automated test plan to use after-suspend test instead of suspend/

for some log attached test cases, I create a new one with prefix after-suspend instead of using flag: also-after-suspend because the log file will be replaced

## Resolved issues

None

## Documentation

- [ ] Automated tests are included for the changed functionality in this PR. If to be merged without tests, please elaborate why.
- [ ] Necessary documentation is provided for the changed functionality in this PR (tests are documentation, too).

## Tests

**before applied**
$ checkbox-cli list-bootstrapped com.canonical.certification::client-cert-desktop-22-04-automated
https://pastebin.canonical.com/p/DKVjqztqZW/

**after applied**
$ checkbox-cli list-bootstrapped com.canonical.certification::client-cert-desktop-22-04-automated
https://pastebin.canonical.com/p/nvnNJn6RZp/

### CODE REVIEW

File: providers/base/units/camera/test-plan.pxu [hunk 1]
```diff
@@ -61,6 +61,16 @@ include:
 bootstrap_include:
  device
 
+id: aftre-suspend-camera-cert-automated
```
Comment (pieqq) [inline providers/base/units/camera/test-plan.pxu:64]: typo: it should be `after-suspend-camera-cert-automated`.

File: providers/base/units/camera/test-plan.pxu [hunk 2]
```diff
@@ -61,6 +61,16 @@ include:
...
+unit: test plan
+_name: Camera tests After Suspend (automated)
+_description: Camera tests After Suspend (automated)
+include:
+ aftre-suspend-camera/detect                                     certification-status=blocker
+ aftre-suspend-camera/multiple-resolution-images_.*              certification-status=blocker
```
Comment (pieqq) [inline providers/base/units/camera/test-plan.pxu:70]: typo in the last two lines, should be `after-suspend...`

File: providers/certification-client/units/client-cert-desktop-22-04.pxu [hunk 3]
```diff
@@ -123,14 +122,27 @@ nested_part:
...
+    after-suspend-networking-cert-automated
+    after-suspend-optical-cert-automated
+    after-suspend-power-management-precheck-cert-automated
+    after-suspend-touchpad-cert-automated
     after-suspend-touchscreen-cert-automated
+    after-suspend-usb-cert-automated
+    after-suspend-usb-automated
     after-suspend-wireless-cert-automated
     after-suspend-bluetooth-cert-automated
+    aftre-suspend-camera-cert-automated
```
Comment (pieqq) [inline providers/certification-client/units/client-cert-desktop-22-04.pxu:145]: Typo: `aftre`.

**Review by pieqq — CHANGES_REQUESTED** [review]
I just did a quick pass and noticed a typo.

Also, it would be good if you could include a `checkbox-cli list-bootstrapped` before the change and after, so that we can compare easily to make sure jobs that should be replaced are, indeed, getting replaced :)

**Comment by weizhenwu** [comment]
> Thanks!
> 
> I just did a quick pass and noticed a typo.
> 
> Also, it would be good if you could include a `checkbox-cli list-bootstrapped` before the change and after, so that we can compare easily to make sure jobs that should be replaced are, indeed, getting replaced :)

Thanks for reviewing!
I've fix the typo and pasted the link of list-bootstrapped output before applying the change.

**Comment by yphus** [comment]
Please rebase your work on top of current main branch

**Pull request merged**